classes/maps.php

```diff
@@ -82,11 +80,8 @@ function set_map_atts( $atts = array() ) {
 			// Do atts for iframes
 			if ( isset( $_GET['sm_map_iframe'] ) ) {
 				// Set atts from GET vars
-				if ( ! empty( $_GET['map_width'] ) ) {
-					$atts['map_width'] = $_GET['map_width'];
-				}
-				if ( ! empty( $_GET['map_height'] ) ) {
-					$atts['map_height'] = $_GET['map_height'];
+				if ( ! empty( $_GET['api_key'] ) ) {
+					$atts['api_key'] = $_GET['api_key'];
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->